apps/site/assets/ts/schedule/components/Map.tsx

```diff
@@ -181,14 +148,15 @@ export default ({
   currentShapes,
   currentStops
 }: Props): ReactElement<HTMLElement> | null => {
-  const [state, dispatch] = useReducer(reducer, {
+  const state = useChannel<DataFromSocketType, DataReducerType>(
     channel,
-    markers: data.markers
-  });
-  useEffect(() => {
-    setupChannels(channel, dispatch);
-    return () => stopChannels(channel);
-  }, [channel, dispatch]);
+    reducer,
+    {
+      channel,
+      markers: []
```

Comment:
question: Is markers supposed to be a empty array and not like the old `data.markers`?

Reply:
one would think so. but doing so added weird large icons to the map.